packages/cli/src/utils.ts

```diff
@@ -32,25 +32,33 @@ export function baseName(fileName: string) {
  * @returns file mtime in ms, or null if the file does not exist or
  * is not a file.
  */
+async function statFile(filePath: PathLike): Promise<Stats | null> {
+  try {
+    return await fsp.stat(filePath);
+  } catch (e) {
+    return null;
+  }
+}
+
```

Comment:
It looks like this function was inserted between another function and its JSDoc comment. The JSDoc for `statFile` is incorrect.

Reply:
Thanks, I'll fix this.